lib/sidekiq-scheduler.rb

```diff
@@ -8,10 +8,10 @@
   config.on(:startup) do
     enabled = Sidekiq::Scheduler.enabled
     scheduler_options = {
-      :scheduler => config.options.fetch(:scheduler, true),
-      :dynamic => config.options.fetch(:dynamic, false),
-      :enabled => enabled.nil? ? true : enabled,
-      :schedule => config.options.fetch(:schedule, nil)
+      scheduler: config.options.fetch(:scheduler, true),
+      dynamic:   config.options.fetch(:dynamic, false),
+      enabled:   enabled.nil? ? true : enabled,
```

Comment:
This is a option that was added recently, and no all configuration have this option, that is way if the option is not present we assume that the scheduler is enabled.